bundles/org.openhab.persistence.jdbc/README.md

```diff
@@ -14,7 +14,7 @@ The following databases are currently supported and tested:
 | [HSQLDB](http://hsqldb.org/)                 | [hsqldb-2.3.3.jar](https://mvnrepository.com/artifact/org.hsqldb/hsqldb) |
 | [MariaDB](https://mariadb.org/)              | [mariadb-java-client-1.4.6.jar](https://mvnrepository.com/artifact/org.mariadb.jdbc/mariadb-java-client) |
 | [MySQL](https://www.mysql.com/)              | [mysql-connector-java-5.1.39.jar](https://mvnrepository.com/artifact/mysql/mysql-connector-java) |
-| [PostgreSQL](https://www.postgresql.org/)    | [postgresql-9.4.1209.jre7.jar](https://mvnrepository.com/artifact/org.postgresql/postgresql) |
+| [PostgreSQL](https://www.postgresql.org/)    | [postgresql-42.3.3.jar](https://mvnrepository.com/artifact/org.postgresql/postgresql) |
 | [SQLite](https://www.sqlite.org/)            | [sqlite-jdbc-3.16.1.jar](https://mvnrepository.com/artifact/org.xerial/sqlite-jdbc) |
 | [TimescaleDB](https://www.timescale.com/)    | [postgresql-9.4.1209.jre7.jar](https://mvnrepository.com/artifact/org.postgresql/postgresql) |
```

Comment:
Just wondering: What about this?

Reply:
Apparently it builds on top of PostgreSQL, so I'll also update that line. :+1: 

https://severalnines.com/product/clustercontrol/timescale-time-series-database-management

> While TimeScaleDB is a relatively new technology, it is built on PostgreSQL to make it both familiar to users as well as be compatible with all the tools and systems that currently support PostgreSQL.

Reply:
If you think it is good now, please merge the PR.
I gave it a test with PostgreSQL and it still worked!